guide/getting-started/open-design.md

```diff
@@ -0,0 +1,112 @@
+
+The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translaters, project managers, and others working together.
+
+##  Open design and Bitcoin
+
+Bitcoin has had the spirit of openness deeply embedded from its humble beginnings. The source code is public, participation in the network is public and unrestricted, and transfering value via the Bitcoin protocoal is permissionless as well. Thousands of projects have copied ideas, techniques and code from Bitcoin to experiment and explore different directions. So it provides a fantastic foundation for designers to also contribute in this open and participatory manner.
+
+One might argue that Bitcoin needs open design in order to stay true to its founding values. The majority of users don't interact directly with a technical protocol, but instead with applications built on top of it. If those applications end up being controlled or manipulated, then it will not matter if the protocol is open. So it is up to the people shaping the open-source application layer to help ensure that the future of Bitcoin remains decentralized and allows for permissionless participation.
+
+## Values
+
+In open design, we... 
+
+- collaborate and don't compete
+- are transparent and share our process and work
+- make it easy for others to use and build on our ideas and work
+- are inclusive and help others
+- are open-minded and flexible 
+- work multidisciplinary, because good results require different perspectives and skillsets
+- continuously learn independently, and from each other
+- rely on our peer group for free and communal validation of quality
+- are accountable for our own actions, and have vested interest in the groups actions and goals
+
+And we value...
+
+- design ethics to make moral and responsible choices
+- human design to create products for real needs and desires
+- usability to create products that allow users to effectively solve their problems
+- accessibility to make products usable by people with the widest possible range of abilities, in the widest possible range of situations
+- aesthetics, which have a highly functional element and also influence how people feel and think
+
+## Getting started
+
+If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
```

Comment:
```suggestion
If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting point is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
```